subsys/bluetooth/services/hids.c

```diff
@@ -738,6 +764,17 @@ hids_input_reports_register(struct bt_gatt_hids *hids_obj,
 	for (size_t i = 0; i < hids_obj->inp_rep_group.cnt; i++) {
 		struct bt_gatt_hids_inp_rep *hids_inp_rep =
 		    &hids_obj->inp_rep_group.reports[i];
+		u8_t rperm = hids_inp_rep->perm & GATT_PERM_READ_MASK;
+		u8_t wperm;
+
+		if (rperm == 0)
```

Comment:
style - missing brackets

Reply:
also in other places - I guess CI will kick in

Reply:
CI did not pick this up it seems.

Hi @carlescufi , don't we anymore have the req to add brackets after if? e.g.
```
if (x) {
   dome();
}
```

Reply:
I suppose it was changed. I did it in the (old?) style before and I get:
```WARNING: braces {} are not necessary for single statement blocks
#224: FILE: subsys/bluetooth/services/hids.c:820:
+		if (0 == (perm & GATT_PERM_READ_MASK)) {
+			perm |= HIDS_GATT_PERM_DEFAULT & GATT_PERM_READ_MASK;
+		}
```

Reply:
ah... ok, I missed an update then

Reply:
we do have the requirement of braces, unless someone has changed something I am not aware of @rakons and @pdunaj. The rationale is just to avoid potential unwanted control flow forks.

Reply:
"potential unwanted control flow forks" comes from poor macro design. Lets improve that and clean trash at input instead of cleaning it at output.
Anyway - I was trying to write using brackets and above you have what I get. So now I am confused.

Reply:
ah... I see we still clarifying this one...

Reply:
that is strange - I checked checkpack and this warning was there from 2015

I guess that script must have failed to recognize that it is analyzing the `if` statement.  There is a code that checks if it is `if`.

Reply:
I think this is a problem in CI, we need to fix it ASAP for other PRs. @thst-nordic will look into it.

Reply:
@rakons , could you fix the style issue so we can merge this change? Warning should not be a problem as we can ignore it.

Reply:
Done

Reply:
it is still incorrect